research/slim/BUILD

```diff
@@ -709,7 +709,7 @@ py_library(
 py_binary(
     name = "train_image_classifier",
     srcs = ["train_image_classifier.py"],
-    paropts = ["--compress"],
+    #paropts = ["--compress"],
```

Comment:
Please remove the changes in slim-- that will also reduce the complexity of this change.